module/stats.py

```diff
@@ -18,21 +20,53 @@ def stats(update: Update, context: CallbackContext):
 def stats_gen(update: Update, context: CallbackContext, days):
     conn = sqlite3.connect('data/DMI_DB.db')
     chat_id = update.message.chat_id
-    query = ""
     text = ""
 
     if days == 0:
         text += "Record Globale:\n"
-        query = "SELECT Type, count(chat_id) FROM stat_list GROUP BY Type ORDER BY Type;"
+        query = "SELECT Type, n FROM (SELECT Type, COUNT(chat_id) as n FROM stat_list GROUP BY Type) ORDER BY n DESC;"
     else:
         text += "Record di "+str(days)+" giorni:\n"
-        query = "SELECT Type, count(chat_id) FROM stat_list WHERE DateCommand > '"+ str(date.today()-timedelta(days=days)) + "' GROUP BY Type ORDER BY Type;"
+        query = "SELECT Type, n FROM (SELECT Type, COUNT(chat_id) as n FROM stat_list WHERE DateCommand > '"+ str(date.today()-timedelta(days=days)) + "' GROUP BY Type) ORDER BY n DESC;"
 
-    for row in conn.execute(query):
-        if str(row[0]) != "leiCheNePensaSignorina" and str(row[0]) != "smonta_portoni" and str(row[0]) != "santino" and str(row[0]) != "bladrim" and str(row[0]) != "prof_sticker":
-            text += str(row[1]) + ": " + str(row[0]) + "\n"
-    context.bot.sendMessage(chat_id=chat_id, text=text)
+    rows = conn.execute(query).fetchall()
     conn.close()
 
+    total = 0
+    n_commands = 0
+    for row in rows:
+        if str(row[0]) not in ("leiCheNePensaSignorina", "smonta_portoni", "santino", "bladrim", "prof_sticker"):
```

Comment:
put `("leiCheNePensaSignorina", "smonta_portoni", ...)` in a separated variable like `easter_egg`, I think it's better